### ISSUE TITLE
JSON error on invoicesApi.Get(invoiceId, parameters1)

### ISSUE THREAD

**Issue by webjv** [zoho/books-dotnet-wrappers#11]
I'm using the sample C# InvoicesApiTest.

If I try to get an invoice using the invoiceId, I get the error: "Newtonsoft.Json.JsonReaderException: 'Unexpected character encountered while parsing value: [. Path 'line_items[0].time_entry_ids', line 67, position 25.'"

The line where the error happend is:
var invoice1 = invoicesApi.Get(invoiceId, parameters1);

Any idea what could be causing it?

**Comment by Tango600** [comment]
Take my fork branch, may be this bug was fixed there.

**Comment by webjv** [comment]
Thank you for the reply @Tango600 but I tested and got the same error message.

**Comment by webjv** [comment]
I tried the "https://books.zoho.com/api/v3/invoices/..." directly in Chrome Browser to test the command. I works but it was returning me a pdf instead o a json... I change the parameter to accept=json and now from the Browser it works fine.
Maybe that is why I got the error in C# but I don't have a way to validate, so I decided to don't use the books-dotnet-wrappers and use RestShap Nuget to have a better control. Now it is working...
